test/unit/db/seeds_test.rb

```diff
@@ -5,7 +5,178 @@
 class SeedsTest < ActiveSupport::TestCase
   disable_transactional_fixtures!
 
-  test 'the seeds do not fail' do
-    assert Rails.application.load_seed
+  def teardown
```

Comment:
it could be defined on one place:

```
env_variables = {
  MASTER_NAME: 'master name' 
}
```

and then:

```
env_variables.each do |key, value|
  ENV[key] = value
```
and then the same for `teardown`

Reply:
to not left any leftovers by any chance..

Reply:
Right 🚀 💪

Reply:
Done 😎 💃